client/reports_test.go

```diff
@@ -172,6 +173,10 @@ func TestPeriodicMonitoring(t *testing.T) {
 			t.Error("server has reports we didn't send", i, report.PowerOutput)
 		}
 	}
+
+	// Event logging output
+	path := filepath.Join(client.staticBaseDir, "status.txt")
+	os.WriteFile(path, []byte(client.DumpStatus()), 0644)
```

Comment:
why?

Reply:
I'm not sure this is going to add value. Probably makes more sense to do a dump after an error is being experienced